README.md

```diff
@@ -103,7 +103,7 @@ def create_changeset(%User{} = user, attrs) do
 end
 ```
 
-### 4: Configuration
+### 4: Configuration to use temporary file functions
```

Comment:
This can be a bit misleading, since the AWS subsection is under this section, but it's not related to temporary files..

I think we should do something like:

### 4: Configuration

Depending on which features you are using, you may need extra configurations:

#### Temporary Files

If you are using `get_temporary_file` or `get_temporary_files`, you need to configure [task_after](https://github.com/OvermindDL1/task_after):

```elixir
config :task_after, global_name: TaskAfter
```

#### S3 Configuration

If you are using the S3 adapter, add this to your configuration file. For more information access the [ex_aws_s3 documentation](https://github.com/ex-aws/ex_aws_s3):

```elixir
config :ex_aws, :s3,
  access_key_id: "key",
  secret_access_key: "secret",
  region: "us-east-1",
  host: "localhost",
  port: "9000",
  scheme: "http://"

config :my_project, :uploads,
  bucket: "uploads",
  region: "us-east-1"
```